DNN Platform/DotNetNuke.Web/InternalServices/EventLogServiceController.cs

```diff
@@ -33,17 +33,16 @@ public HttpResponseMessage GetLogDetails(string guid)
 
             try
             {
-                var logInfo = new LogInfo { LogGUID = guid };
-                logInfo = EventLogController.Instance.GetSingleLog(logInfo, LoggingProvider.ReturnType.LogInfoObjects) as LogInfo;
-                if (logInfo == null)
+                var log = EventLogController.Instance.GetLog(guid);
+                if (log == null || !(log is LogInfo))
                 {
                     return this.Request.CreateResponse(HttpStatusCode.BadRequest);
                 }
 
                 return this.Request.CreateResponse(HttpStatusCode.OK, new
                 {
                     Title = Localization.GetSafeJSString("CriticalError.Error", Localization.SharedResourceFile),
-                    Content = this.GetPropertiesText(logInfo),
+                    Content = this.GetPropertiesText((LogInfo)log),
```

Comment:
```suggestion
                    Content = this.GetPropertiesText(logInfo),
```

Reply:
This makes more sense, sorry about the confusion. We are going to save 1 object allocation by doing it in 1 line.

I'll make the necessary changes as you originally recommended.